experimental/packages/opentelemetry-exporter-prometheus/src/PrometheusSerializer.ts

```diff
@@ -261,7 +261,7 @@ export class PrometheusSerializer {
       results += stringify(
         name + '_' + key,
         attributes,
-        value[key],
+        value[key] ?? 0,
```

Comment:
I'd find coalescing the sum value to 0 can be confusing. Would it be possible to not output this value?

Reply:
I also though it was weird, I did that as this seems to be the behavior of the exporter in collector-contrib. Not outputting the value sounds like the better approach, I adopted the idea in [695aeb8](https://github.com/open-telemetry/opentelemetry-js/pull/3102/commits/695aeb8fa798cc2db5c9bb8df50db0145fab322c). :slightly_smiling_face: 

I'll also open an issue in collector contrib to see what people there think. :slightly_smiling_face:

Reply:
Opened a discussion on collector-contrib here: https://github.com/open-telemetry/opentelemetry-collector-contrib/discussions/12646 :slightly_smiling_face: